channel/src/lib.rs

```diff
@@ -90,42 +101,16 @@ impl Channel {
     pub fn add_message(&mut self, message: Message) {
         self.messages.push(message);
     }
-
-    pub fn subs(&self) -> BTreeSet<ActorId> {
-        self.subscribers.clone()
-    }
 }
 
-#[gstd::async_init]
-async fn init() {
-    let channel_init: ChannelInit = msg::load().expect("Unable to decode ChannelInit");
-
+#[no_mangle]
+unsafe extern "C" fn init() {
```

Comment:
We prefer not to use `unsafe`.
```suggestion
extern "C" fn init() {
```